spec/fixtures/exports/general_needs_log_24_25.xml

```diff
@@ -144,8 +144,8 @@
     <discarded_at/>
     <creation_method>1</creation_method>
     <duplicate_set_id/>
-    <nationality_all/>
```

Comment:
due to reordering of fields - I _think_ this is fine but haven't worked on export much so not sure if changing this is a problem

Reply:
I don't have any practical experience of this either, but it would be extremely odd for this behaviour to matter.